### PR TITLE
Media resources: make hashing client IP a setting

### PR DESCRIPTION
**Pull request by jackvanb**
Creates an environment variable to turn on/off including the client IP in the hash generation for BruinCast and VidoeReserves.

### CODE REVIEW

File: src/server/api/medias/index.js [hunk 1]
```diff
@@ -307,6 +318,7 @@ router.get('/url', (req, res) => {
   const now = new Date();
   const start = Math.round(now.getTime() / 1000);
   const end = start + parseInt(VALIDITY);
+  clientIP = hashClientIP ? clientIP : null;
```
Comment (rlorenzo) [inline src/server/api/medias/index.js:321]: @jackvanb Why not simplify this to:

`const clientIP = hashClientIP ? requestIp.getClientIp(req) : null;`

Then you don't need line 278

**Review by rlorenzo — CHANGES_REQUESTED** [review]

**Comment by rlorenzo** [comment]
@jackvanb Also squash your commits so that you can pass the "Check Commit Message" Github action check

File: src/server/api/medias/index.js [hunk 2]
```diff
@@ -293,11 +296,19 @@ router.get('/url', (req, res) => {
       return res.status(400).send(new Error('Incorrect format for quarter'));
     }
 
+    if (BRUINCAST_HASH_CLIENT_IP === 'true') {
```
Comment (rlorenzo) [inline src/server/api/medias/index.js:299]: @jackvanb So boolean values in .env are converted to strings?

**Comment by rlorenzo** [comment]
@jackvanb Can you also add a Jest test to verify this code change?

File: src/server/api/medias/index.js (hunk 2 above)
Comment (jackvanb) [reply src/server/api/medias/index.js:299]: All .env values are treated as strings. (https://github.com/motdotla/dotenv/issues/51)

This seems to be the clearest way to express the boolean logic.

**Comment by jackvanb** [comment]
> @jackvanb Can you also add a Jest test to verify this code change?

Currently, there aren't any tests for the API routes. Should this be a separate ticket to write tests for the routes?

File: src/server/api/medias/index.js (hunk 1 above)
Comment (jackvanb) [reply src/server/api/medias/index.js]: Done

**Pull request merged**